app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -170,19 +170,13 @@ def raw_connect(api_key, pcloud_guid)
       end
 
       require "ibm_cloud_resource_controller"
-      api_client = IbmCloudResourceController::ApiClient.new
-      api_client.config.api_key        = {"Authorization" => token.access_token}
-      api_client.config.api_key_prefix = {"Authorization" => token.token_type}
-      api_client.config.access_token   = {"Authorization" => token.access_token}
-      api_client.config.logger         = $ibm_cloud_log
-
-      resource_instances_api = IbmCloudResourceController::ResourceInstancesApi.new(api_client)
+      authenticator = IbmCloudResourceController::Authenticators::BearerTokenAuthenticator.new(:bearer_token => token.access_token)
+      resource_controller_api = IbmCloudResourceController::ResourceControllerV2.new(:authenticator => authenticator)
 
       begin
-        power_iaas_service = resource_instances_api.get_resource_instance(pcloud_guid)
-      rescue IbmCloudResourceController::ApiError => err
-        error_message = JSON.parse(err.response_body)["message"]
-        _log.error("GUID resource lookup failed: #{err.code} #{error_message}")
+        power_iaas_service = resource_controller_api.get_resource_instance(:id => pcloud_guid).result
+      rescue IbmCloudResourceController::ApiException => err
+        _log.error("GUID resource lookup failed: #{err.code} #{err.error}")
         raise MiqException::MiqInvalidCredentialsError, error_message
```

Comment:
One more pesky `error_message` here

Reply:
Thanks, trying to do too many things at once haha